scripts/analysis.py

```diff
@@ -510,85 +343,195 @@ def get_swu_dict(cursor):
 
     Returns
     -------
-    dictionary of swu timeseries
+    swu_dict: dictionary
+        dictionary with key: Enrichment (facility number),
+        value: swu timeseries
     """
-
-    cur = cursor
     swu_dict = collections.OrderedDict()
-    agentid = get_agent_ids(cur, 'Enrichment')
-    init_year, init_month, duration, timestep = get_sim_time_duration(cur)
+    agentid = get_agent_ids(cursor, 'Enrichment')
+    init_year, init_month, duration, timestep = get_sim_time_duration(cursor)
     facility_num = 1
     for num in agentid:
-        swu_data = cur.execute('SELECT time, value FROM timeseriesenrichmentswu \
-                               WHERE agentid = ' + str(num)).fetchall()
-        swu_timeseries = get_timeseries(swu_data, duration, 1)
+        swu_data = cursor.execute('SELECT time, value '
+                                  'FROM timeseriesenrichmentswu '
+                                  'WHERE agentid = ' + str(num)).fetchall()
+        swu_timeseries = get_timeseries(swu_data, duration, 1, 'TRUE')
         swu_dict['Enrichment' + str(facility_num)] = swu_timeseries
         facility_num += 1
 
     return swu_dict
 
 
-def get_timeseries(list, duration, multiplyby):
-    """ returns a timeseries list of a given data
+def get_power_dict(cursor):
+    """ Gets dictionary of power capcity and number of reactors
+    by calling capacity_calc
 
     Parameters
     ----------
-    list: list
-        list of data to be created into timeseries
-        list[0] = time
-        list[1] = value, quantity
-    duration: int
-        duration of the simulation
-    multiplyby: int
-        integer to multiply the value in the list by
+    cursor: sqlite cursor
+        sqlite cursor
 
     Returns
-    -------
-    timeseries list of data
+    ------
+    capacity_calc: function
+        calls a function
     """
+    init_year, init_month, duration, timestep = get_sim_time_duration(cursor)
+    # get power cap values
+    governments = cursor.execute('SELECT prototype, agentid FROM agententry '
```

Comment:
Can you explain why it should be a function please?
I can see that it might make the code more modular but the function would simply be one line returning `cursor.execute(...)`. Furthermore, this sqlite query is only called within this function with no plans to add any other functions at this moment in time.
Thank you.